api/externals/repository/source_repository.go

```diff
@@ -27,30 +28,44 @@ func NewSourceRepository(c db.Client, ac abstract.Crud) SourceRepository {
 
 // 全件取得
 func (sr *sourceRepository) All(c context.Context) (*sql.Rows, error) {
-	query := "select * from sources"
+	query := "SELECT * FROM sources"
 	return sr.abstract.Read(c, query)
```

Comment:
```suggestion
	return sr.crud.Read(c, query)
```
ここ自分が過去実装忘れてたんだけど、ここcrud.Readにした方がいい感じすね

11~14行目のsourceRepositoryの構造体のフィールド名を以下にする必要があります。
```
type sourceRepository struct {
	client   db.Client
  crud abstract.Crud
}
```